### DISCUSSION TITLE
How to avoid installation of AMDNoiseSuppression?

### DISCUSSION THREAD

**Discussion by nilssohn** [GSDragoon/RadeonSoftwareSlimmer#43] [Q&A] [answered]
Hi,

I've been asking this before, maybe slightly offtopic there:
https://github.com/GSDragoon/RadeonSoftwareSlimmer/discussions/37#discussioncomment-6290268

So again: How can RSS be configured in a way that doesn't install at all (or remove after installation) AMDNoiseSuppression in Windows Autostart? I haven't yet been able to accomplish this.

Let me point out that I am currently writing an article about this great tool for a german hardware gaming magazin (print and online). It would be great if there was a solution for this small issue I can mention there.

**Comment by GSDragoon** [comment, marked as answer]
> Hi,
> 
> I've been asking this before, maybe slightly offtopic there: [#37 (comment)](https://github.com/GSDragoon/RadeonSoftwareSlimmer/discussions/37#discussioncomment-6290268)
> 
> So again: How can RSS be configured in a way that doesn't install at all (or remove after installation) AMDNoiseSuppression in Windows Autostart? I haven't yet been able to accomplish this.
> 
> Let me point out that I am currently writing an article about this great tool for a german hardware gaming magazin (print and online). It would be great if there was a solution for this small issue I can mention there.

You can prevent AMD Noise Suppression from being installed by unchecking it under `Display Driver Components`. If it's already installed, you might need a clean install to remove it.

![image](https://github.com/GSDragoon/RadeonSoftwareSlimmer/assets/37422356/5812803f-182e-4ba9-8d23-11b30a800c16)

I'm not sure where the exactly startup entry is created from, but it should be harmless if not installed. The process it tries to run doesn't exist. It tries to run `C:\Windows\system32\AMD\ANR\AMDNoiseSuppression.exe`, but `C:\Windows\system32\AMD\ANR` does not exist because the component is not installed.

It's in the registry under `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run`. You can disable it with Task Manager, built into Windows, or a tool like [Autoruns](https://learn.microsoft.com/en-us/sysinternals/downloads/autoruns). It should stay disabled across installs, as long as you aren't doing a clean install.
